data/binding/binditems.go

```diff
@@ -10,16 +10,22 @@ type Bool interface {
 	Set(bool)
 }
 
+// BoolPointer supports binding a bool value in a Fyne application
+type BoolPointer interface {
+	Bool
+	DataPointer
+}
+
 // NewBool returns a bindable bool value that is managed internally.
 func NewBool() Bool {
```

Comment:
If this returned `{{ .Name }}Pointer` then the cast on line 28 wouldn't be needed

Reply:
Doing this would expose `Reload` on the standard `Bool` type which I am working very hard to avoid.
The pointer is internal and auto-generated, not sure we need to worry.